pepper-apis/src/main/java/org/broadinstitute/ddp/json/invitation/Invitation.java

```diff
@@ -0,0 +1,77 @@
+package org.broadinstitute.ddp.json.invitation;
+
+import java.time.Instant;
+
+import javax.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+import org.broadinstitute.ddp.db.dto.InvitationDto;
+import org.broadinstitute.ddp.model.invitation.InvitationType;
+
+public class Invitation {
+
+    @SerializedName("invitationType")
+    private InvitationType invitationType;
+    @SerializedName("invitationId")
+    private String invitationGuid;
```

Comment:
We have some annotations that convert fields with value to date format.  Since we store date/timestamps in our database using different data types, I think they serve the purpose of making sure that our date serialization in our APIs is consistent. Consider using of our GSON adapters or creating a new one if we don't have one for `Instant`

Reply:
I didn't know what/where to look for for these magical gson adapters so I just went with a `String`. I'll take a second look.

Reply:
I have added a new `InstantStringAdapter` class to handle this.